packages/engine/src/worker/runner/javascript/batch.js

```diff
@@ -1,285 +1,282 @@
 // noinspection BadExpressionStatementJS
+import { arrow } from "./src/worker/runner/javascript/apache-arrow-bundle.js";
+import * as hash_util from "./src/worker/runner/javascript/hash_util.js";
 
-(arrow, hash_util) => {
-  "use strict";
```

Comment:
Will using modules rather than evals ensure, that we still run javascript in strict mode?

Reply:
Yes, modules use strict mode by default.
[source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts)
> modules use [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) automatically